File: ApiSuvesaPos/Negocio/Modelo/cxc/apartados/apartado_detalle.vb
Namespace Modelo
    Public Class apartado_detalle
        Public Property Id_detalle As Long
        Public Property Id_Apartado As Long
        Public Property Codigo As String
        Public Property Descripcion As String
        Public Property Cantidad As Integer
        Public Property Precio_Costo As Decimal
        Public Property Precio_Base As Decimal
        Public Property Precio_Flete As Decimal
        Public Property Precio_Otros As Decimal
        Public Property Precio_Unit As Decimal
        Public Property Impuesto As Decimal
        Public Property Monto_Impuesto As Decimal
        Public Property SubtotalGravado As Decimal
        Public Property SubTotalExcento As Decimal
        Public Property SubTotal As Decimal
        Public Property Monto_descuento As Decimal
        Public Property Descuento As Decimal
        Public Property Max_Descuento As Decimal
        Public Property Tipo_Cambio_ValorCompra As Decimal
        Public Property Cod_MonedaVenta As Integer
        Public Property IdBodega As Long
        Public Property cod_articulo As String

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/apartados/apartado_detalle.cs
namespace Modelo
{
    public class apartado_detalle
    {
        public long Id_detalle { get; set; }
        public long Id_Apartado { get; set; }
        public string Codigo { get; set; }
        public string Descripcion { get; set; }
        public int Cantidad { get; set; }
        public decimal Precio_Costo { get; set; }
        public decimal Precio_Base { get; set; }
        public decimal Precio_Flete { get; set; }
        public decimal Precio_Otros { get; set; }
        public decimal Precio_Unit { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Monto_Impuesto { get; set; }
        public decimal SubtotalGravado { get; set; }
        public decimal SubTotalExcento { get; set; }
        public decimal SubTotal { get; set; }
        public decimal Monto_descuento { get; set; }
        public decimal Descuento { get; set; }
        public decimal Max_Descuento { get; set; }
        public decimal Tipo_Cambio_ValorCompra { get; set; }
        public int Cod_MonedaVenta { get; set; }
        public long IdBodega { get; set; }
        public string cod_articulo { get; set; }
    }
}

